Change LED[0] control mode to On - Link, Off - No Link

diff --git a/drivers/net/ethernet/intel/igb/e1000_phy.c b/drivers/net/ethernet/intel/igb/e1000_phy.c
index 413025b..2629529 100644
--- a/drivers/net/ethernet/intel/igb/e1000_phy.c
+++ b/drivers/net/ethernet/intel/igb/e1000_phy.c
@@ -2281,6 +2281,16 @@ s32 igb_initialize_M88E1512_phy(struct e1000_hw *hw)
 	if (ret_val)
 		goto out;
 
+	/* Switch to PHY page 0x3. */
+	ret_val = phy->ops.write_reg(hw, E1000_M88E1543_PAGE_ADDR, 0x3);
+	if (ret_val)
+		goto out;
+
+	/* Change LED[0] control mode to On - Link, Off - No Link */
+	ret_val = phy->ops.write_reg(hw, E1000_M88E1512_CFG_REG_1, 0x1010);
+	if (ret_val)
+		goto out;
+
 	/* Return the PHY to page 0. */
 	ret_val = phy->ops.write_reg(hw, E1000_M88E1543_PAGE_ADDR, 0);
 	if (ret_val)
